argusclient/model.py

```diff
@@ -488,6 +488,39 @@ def __init__(self, name, notifierName=None, metricsToAnnotate=None, **kwargs):
                                            **kwargs)
 
 
+class Derivative(BaseEncodable):
+    """
+    Represents a Derivative object in Argus.
+    ** Required parameteres to the connstructor:**
+    :param name: The name of the derivative
+    :type name: str
+    :param sourceExpression: The argus query for source metric
+    :type sourceExpression: str
+    :param derivedScope: The scope for derived metric
+    :type derivedScope: str
+    :param derivativeInterval: Interval at which derivative job needs to run
+    :type derivativeInterval: str
+
+    ** Optional parameteres to the connstructor:**
+    :param derivedMetric: The metric name for derived metric
+    :type derivedMetric: str
+    :param derivativeTagOptions: The tag modifiers for derived metric
+    :type derivativeTagOptions: str
+    :param enabled: Denotes whether job is enabled or not
+    :type enabled: bool
+    :param alertIfNoData: Denotes whether to alert user on no data or not.
+    :type alertIfNoData: str
+    """
+
+    id_fields = ("sourceExpression", "derivedScope", "derivativeInterval",)
+
+    def __init__(self, name, sourceExpression, derivedScope, derivativeInterval, **kwargs):
+        super(Derivative, self).__init__(name=name, sourceExpression = sourceExpression, derivedScope = derivedScope,
+                                            derivativeInterval = derivativeInterval, **kwargs)
+
+
+
+
```

Comment:
Per [PEP8](https://www.python.org/dev/peps/pep-0008/), need to have only [two blank lines between top level entities](https://www.python.org/dev/peps/pep-0008/#blank-lines).